docs/source/xrpl.asyncio.account.rst

```diff
@@ -1,27 +1,5 @@
-xrpl.asyncio.account package
-============================
-
-Submodules
-----------
-
-xrpl.asyncio.account.main module
---------------------------------
-
-.. automodule:: xrpl.asyncio.account.main
-   :members:
-   :undoc-members:
-   :show-inheritance:
-
-xrpl.asyncio.account.transaction\_history module
-------------------------------------------------
-
-.. automodule:: xrpl.asyncio.account.transaction_history
-   :members:
-   :undoc-members:
-   :show-inheritance:
-
-Module contents
----------------
+XRPL Async Account Methods
```

Comment:
Those `.rst` files are generated automatically, aren't they?

Reply:
Yes, but the auto-generated versions don't really look that nice and have a lot of repeated info due to how we did imports